NOTES.UNIX

```diff
@@ -28,3 +28,12 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         -Wl,-rpath,'$(LIBRPATH)'
+
+ On most GNU systems, a better choice is to use the new dtags, like this:
```

Comment:
I don't think "GNU" is the right thing here ...

Reply:
What is, then?  This seems to be very much tied in to systems that use the GNU toolchain all the way down to ld.so...  or is this Linux only?  Please advise

Reply:
If [this wikipedia page](https://en.wikipedia.org/wiki/Rpath) is trustworthy, it does seem that "GNU" is the right thing.

Reply:
Maybe it's better to start the sentence with "With GNU ld.so, a better choice ..."?

Reply:
It is the "GNU system" I think is what was triggering the comment. It is about the combination of the compile time linker and runtime linker.